code/modules/surgery/organs/internal/eyes/_eyes.dm

```diff
@@ -416,6 +416,7 @@
 /obj/item/organ/internal/eyes/robotic/glow/Destroy()
 	. = ..()
 	deactivate(close_ui = TRUE)
+	eye.moveToNullspace()
```

Comment:
Why's this necessary, movable atoms automatically move themselves to nullspace when they get `qdel`'d.